components/VerticalCards/Card.tsx

```diff
@@ -0,0 +1,105 @@
+import { useState, useEffect } from "react";
+import { VerticalCardData, Fade } from "../../types";
+import { fadeElement } from "../../lib/utils";
+
+interface CardProps extends VerticalCardData {
+  selectedIdx: number;
+  idx: number;
+  onToggle(idx: number): any;
+}
+
+const Card = ({
+  selectedIdx,
+  idx,
+  onToggle,
+  title,
+  blurb,
+  body,
+}: CardProps) => {
+  const [isHovered, setHovered] = useState(false);
+  const [content, setContent] = useState(blurb);
+
+  const selected = selectedIdx == idx;
+  const somethingIsSelected = selectedIdx !== -1;
+
+  const handleMouseEnter = () => {
+    setHovered(true);
+  };
+
+  const handleMouseLeave = () => {
+    setHovered(false);
+  };
+
+  const containerHeight = () => {
+    if (!selected && !somethingIsSelected) return "h-[30vw] xl:h-[10vw] opacity-100 overflow-hidden";
+    else if (!selected && somethingIsSelected) return "h-0 z-0 opacity-0";
+    else return "h-[80vw] xl:h-[40vw] z-10 m-5 overflow-hidden";
+  };
+
+  const containerBackground = () => {
+    if (isHovered) return "bg-wato-blue";
```

Comment:
Background should be blue when selected as well.

Change `isHovered` to `isHovered || selected`